### PR TITLE
Search for a product. https://app.clickup.com/t/8689p8y0m

### PR DESCRIPTION
**Pull request by AlexMotornyi92**
Objective: Verify the search functionality on the store web page.

Description:

Navigate to Store Page: Open the store web page at "https://askomdch.com/store/".
Enter Search Query: Type "Shoes" into the search field.
Initiate Search: Click the "Search" button to start the search.
Wait for Results: A 3-second delay is introduced to allow time for the search results to load.
Verify Results: Check that the search results header contains the word "Shoes".
Expected Result: The search results header should include the word "Shoes". If this is confirmed, the test is considered successful.

### CODE REVIEW

File: src/test/java/FirstTest.java [hunk 1]
```diff
@@ -789,4 +789,17 @@ public void testStoreAddItemToMyCart(){
 
         Assert.assertTrue(amount.isDisplayed());
     }
-}
+
+    @Test
+    public void UseSearchOption() throws InterruptedException {
```
Comment (ElenaDetusheva) [inline src/test/java/FirstTest.java:794]: название класса для тестов начинается с test

**Review by ElenaDetusheva — COMMENTED** [review]
название класса для тестов начинается с test

**Review by ElenaDetusheva — COMMENTED** [review]
А зачем вы пишите throws InterruptedException, если в тесте у вас нигде ошибки не выбрасываются?

File: src/test/java/FirstTest.java (hunk 1 above)
Comment (AlexMotornyi92) [reply src/test/java/FirstTest.java]: Thanks! Right you are. I've fixed name of class, but my code doesn't work without throws InterruptedException , namely in part of Thread.sleep

**Review by w3code — APPROVED** [review]

**Review by ElenaDetusheva — COMMENTED** [review]
LGTM

**Pull request merged**